commands/init.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import datetime
+
+from config import config_context
+
+help="Interactively initialize a new configuration"
+
+logger = logging.getLogger(__name__)
+
+def prompt(explanation, default=None):
+    prompt_string = ''
+    if default is not None:
+        prompt_string = "{} (default: {}): ".format(explanation, default)
+    else:
+        prompt_string = "{}: ".format(explanation)
+    value = input(prompt_string)
+
+    if value == '':
+        if default is not None:
+            return default
+
+        return prompt(explanation, default)
+
+    return value
+
+def guess_semester():
+    now = datetime.datetime.now()
+    if now.month < 5:
+        semester = 'SP'
+    elif now.month < 8:
+        semester = 'SS'
+    else:
+        semester = 'FS'
+
+    return "{}-{}".format(now.year, semester)
+
+
+@config_context
+def init(conf, args):
+    conf['gitlab-host'] = "https://{}".format(prompt("Gitlab server to use", "gitlab.com"))
```

Comment:
We've tried our best to keep assigner institution-agnostic but I could be easily persuaded to set the default values to something useful for everyone using this tool right now.

@michaelwisely ?